.github/workflows/auto-label-issue.yml

```
@@ -0,0 +1,19 @@
+name: Auto Label issues
+on:
+  issues:
+    types:
+      - reopened
+      - opened
+jobs:
+  label_issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Label issues
+        uses: andymckay/labeler@5c59dabdfd4dd5bd9c6e6d255b01b9d764af4414
```

Please change the version to reference the latest release tag instead of a commit hash.

```suggestion
        uses: andymckay/labeler@1.0.3
```